sdk/digitaltwins/digital-twins-core/README.md

```diff
@@ -9,118 +9,80 @@ This package contains an isomorphic SDK for Azure Digital Twins API to provide a
-```
+#### CORS
 
-##### Sample code
-
-See https://github.com/Azure/ms-rest-browserauth to learn how to authenticate to Azure in the browser.
-
-- index.html
-
-```html
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <title>@azure/digital-twins-core sample</title>
-    <script src="node_modules/@azure/core-http/dist/coreHttp.browser.js"></script>
-    <script src="node_modules/@azure/core-arm/dist/coreArm.js"></script>
-    <script src="node_modules/@azure/ms-rest-browserauth/dist/msAuth.js"></script>
-    <script src="node_modules/@azure/digital-twins-core/dist/digital-twins-core.js"></script>
-    <script type="text/javascript">
-      const subscriptionId = "<Subscription_Id>";
-      const authManager = new msAuth.AuthManager({
-        clientId: "<client id for your Azure AD app>",
-        tenant: "<optional tenant for your organization>"
-      });
-      authManager.finalizeLogin().then((res) => {
-        if (!res.isLoggedIn) {
-          // may cause redirects
-          authManager.login();
-        }
-        const client = new Azure.Digitaltwins.AzureDigitalTwinsAPI(res.creds, subscriptionId);
-        const dependenciesFor = ["testdependenciesFor"];
-        const includeModelDefinition = true;
-        const maxItemCount = 1;
-        client.digitalTwinModels
-          .list(dependenciesFor, includeModelDefinition, maxItemCount)
-          .then((result) => {
-            console.log("The result is:");
-            console.log(result);
-          })
-          .catch((err) => {
-            console.log("An error occurred:");
-            console.error(err);
-          });
-      });
-    </script>
-  </head>
-  <body></body>
-</html
-```
+Azure Digital Twins doesn't currently support Cross-Origin Resource Sharing (CORS). As a result, this library cannot be used to make direct calls to the template service from a browser. Please refer to [this document](https://github.com/Azure/azure-sdk-for-js/blob/master/samples/cors/ts/README.md) for guidance.
 
 ## Key concepts
 
-Azure Digital Twins Preview is an Azure IoT service that creates comprehensive models of the physical environment. It can create spatial intelligence graphs to model the relationships and interactions between people, spaces, and devices.
+### Azure Digital Twins
+
+Azure Digital Twins is an Azure IoT service that creates comprehensive models of the physical environment. It can create spatial intelligence graphs to model the relationships and interactions between people, spaces, and devices.
 You can learn more about Azure Digital Twins by visiting [Azure Digital Twins Documentation](https://docs.microsoft.com/azure/digital-twins/)
 
+### `DigitalTwinsClient`
+
+`DigitalTwinsClient` is the client object that users of this library use to manage their Azure Digital Twins instance.
+
 ## Examples
 
-Please take a look at the Readme file in
+### TypeScript example for authentication, client creation and listing models in an Azure Digital Twins instance.
+
+```typescript
```

Comment:
We encourage the use of JS in our samples in the readme file rather than TS as it requires less set up to copy - paste and run. Also is listing of the models most common scenario one would encounter when using this package? If not, I would recommend updating the code snippet to show something that would be great for a person just getting started with the package